src/Swarm/TUI/Controller.hs

```diff
@@ -166,6 +175,13 @@ getTutorials sc = case M.lookup "Tutorials" (scMap sc) of
 advanceMenu :: Menu -> Menu
 advanceMenu = _NewGameMenu . lens NE.head (\(_ :| t) a -> a :| t) %~ BL.listMoveDown
 
+handleMainMessagesEvent :: BrickEvent Name AppEvent -> EventM Name AppState ()
+handleMainMessagesEvent = \case
+  Key V.KEsc -> uiState . uiMenu .= MainMenu (mainMenu Messages)
+  CharKey 'q' -> halt
+  ControlKey 'q' -> halt
```

Comment:
Sure, I guess it will feel more natural to make <kbd>Q</kbd> return to the main menu.

After all, it looks the same as messages in-game. :sweat_smile: